Tests/Brokerages/InteractiveBrokers/InteractiveBrokersBrokerageDataQueueHandlerTest.cs

```diff
@@ -23,7 +23,7 @@
 namespace QuantConnect.Tests.Brokerages.InteractiveBrokers
 {
     [TestFixture]
-    [Ignore("These tests require the IBController and IB TraderWorkstation to be installed.")]
+   // [Ignore("These tests require the IBController and IB TraderWorkstation to be installed.")]
```

Comment:
I had commented it for running the tests locally and forgot to put it back in. My bad.